pkgs/by-name/su/surge-XT/package.nix

```
@@ -79,6 +91,7 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [
```

Another change that's worth doing while we are here.

```suggestion
meta = {
description = "LV2 & VST3 synthesizer plug-in (previously released as Vember Audio Surge)";
homepage = "https://surge-synthesizer.github.io";
@@ -79,6 +91,7 @@ stdenv.mkDerivation rec {
maintainers = with lib.maintainers; [
```